lib/active_model_serializers/adapter/json_api/relationship.rb

```diff
@@ -6,21 +6,22 @@ class Relationship
         # {http://jsonapi.org/format/#document-links Document Links}
         # {http://jsonapi.org/format/#document-resource-object-linkage Document Resource Relationship Linkage}
         # {http://jsonapi.org/format/#document-meta Docment Meta}
-        def initialize(parent_serializer, serializer, options = {}, links = {}, meta = nil)
+        def initialize(parent_serializer, serializer, serializable_resource_options, args = {})
```

Comment:
I'm not a huge fan of that, honestly. Even though it may be bad no have methods with lots of parameters all over the place, I think that in this case in would help out clarity.